pkg/rancher-desktop/backend/lima.ts

```diff
@@ -752,13 +752,18 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
     const VMNETDir = path.join(VMNET_DIR, 'bin');
     const pathList = (process.env.PATH || '').split(path.delimiter);
     const newPath = [binDir, VMNETDir].concat(...pathList).filter(x => x);
+    const env = structuredClone(process.env);
 
-    // LD_LIBRARY_PATH is set for running from an extracted Linux zip file, that includes QEMU,
-    // to make sure QEMU dependencies are loaded from the bundled lib directory,
-    // LD_LIBRARY_PATH is ignored on macOS.
-    return {
-      ...process.env, LIMA_HOME: paths.lima, LD_LIBRARY_PATH: libDir, PATH: newPath.join(path.delimiter),
-    };
+    env['LIMA_HOME'] = paths.lima;
+    env['PATH'] = newPath.join(path.delimiter);
+    if (!env['APPIMAGE']) {
+      // LD_LIBRARY_PATH is set for running from an extracted Linux zip file, that includes QEMU,
+      // to make sure QEMU dependencies are loaded from the bundled lib directory,
+      // LD_LIBRARY_PATH is ignored on macOS.
```

Comment:
And this is also irrelevant on rpm/deb (because we don't ship that directory at all), right? Please add that to the comment.

Also, if `LD_LIBRARY_PATH` is set, perhaps we should append to it instead of clobbering?

Possible rephrasing (I realize you just moved it):
```
Override LD_LIBRARY_PATH to pick up the qemu libraries.
- on macOS, this is not used. The macOS dynamic linker uses LYLD_ prefixed variables.
- on packaged (rpm/deb) builds, we do not ship this directory, so it does nothing.
- we skip this on AppImage builds because ?????
- this only takes effect on builds extracted from a Linux zip file (which includes a bundled
  QEMU) to make sure QEMU dependencies are loaded from the bundled lib directory.
```